core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -2873,34 +2853,27 @@ object ZIO extends ZIOCompanionPlatformSpecific {
     }
 
   /**
-   * Applies the function `f` to each element of the `Chunk[A]` and runs
-   * produced effects in parallel, discarding the results.
-   *
-   * For a sequential version of this method, see `foreach_`.
-   */
-  final def foreachPar_[R, E, A](as: Chunk[A])(f: A => ZIO[R, E, Any]): ZIO[R, E, Unit] =
-    as.mapMPar_(f)
-
-  /**
-   * Applies the functionw `f` to each element of the `Iterable[A]` in parallel,
-   * and returns the results in a new `List[B]`.
+   * Applies the functionw `f` to each element of the `Collection[A]` in parallel,
```

Comment:
Not from your PR but I just noticed a small typo here.

Reply:
Thanks! Fixed now!